composer.json

```diff
@@ -2,56 +2,81 @@
+    "keywords": [
+        "sylius",
+        "sylius-plugin"
+    ],
     "license": "MIT",
     "require": {
-        "php": "^7.4 || ^8.0",
-        "sylius/sylius": "^1.6.0"
+        "php": "^7.4",
+        "sylius/sylius": "~1.8.0 || ~1.9.0"
+    },
+    "conflict": {
+        "api-platform/core": "^2.6",
+        "doctrine/dbal": ">2.13"
     },
     "require-dev": {
-        "lchrusciel/api-test-case": "^5.0",
-        "matthiasnoback/symfony-dependency-injection-test": "^4.0",
+        "behat/behat": "^3.6.1",
+        "behat/mink-selenium2-driver": "^1.4",
+        "dmore/behat-chrome-extension": "^1.3",
+        "dmore/chrome-mink-driver": "^2.7",
+        "friends-of-behat/mink": "^1.8",
+        "friends-of-behat/mink-browserkit-driver": "^1.4",
+        "friends-of-behat/mink-extension": "^2.4",
+        "friends-of-behat/page-object-extension": "^0.3",
+        "friends-of-behat/suite-settings-extension": "^1.0",
+        "friends-of-behat/symfony-extension": "^2.1",
+        "friends-of-behat/variadic-extension": "^1.3",
+        "friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev",
+        "lakion/mink-debug-extension": "^2.0.0",
+        "lchrusciel/api-test-case": "^5.1",
+        "matthiasnoback/symfony-dependency-injection-test": "^4.3",
         "phpspec/phpspec": "^7.0",
-        "phpstan/phpstan": "^0.12.82",
-        "phpstan/phpstan-strict-rules": "^0.12.9",
-        "phpstan/phpstan-webmozart-assert": "^0.12.12",
-        "phpunit/phpunit": "^9",
+        "phpstan/extension-installer": "^1.0",
+        "phpstan/phpstan": "0.12.88",
+        "phpstan/phpstan-doctrine": "0.12.37",
+        "phpstan/phpstan-strict-rules": "^0.12.0",
+        "phpstan/phpstan-symfony": "^0.12",
+        "phpstan/phpstan-webmozart-assert": "0.12.12",
+        "phpunit/phpunit": "^9.5",
+        "sensiolabs/security-checker": "^6.0",
         "sylius-labs/coding-standard": "^4.0",
-        "symfony/browser-kit": "^5.2",
-        "symfony/debug-bundle": "^5.1",
-        "symfony/dotenv": "^4.2",
-        "symfony/intl": "^4.1",
-        "symfony/web-profiler-bundle": "^5.0",
-        "symfony/web-server-bundle": "^4.1"
-    },
-    "autoload": {
-        "psr-4": {
-            "SitemapPlugin\\": "src/"
-        }
+        "symfony/browser-kit": "^4.4 || ^5.2",
+        "symfony/debug-bundle": "^4.4 || ^5.2",
+        "symfony/dotenv": "^4.4 || ^5.2",
+        "symfony/intl": "^4.4 || ^5.2",
+        "symfony/web-profiler-bundle": "^4.4 || ^5.2",
+        "vimeo/psalm": "4.6.4"
     },
-    "prefer-stable": true,
     "config": {
-        "sort-packages": true
-    },
-    "autoload-dev": {
-        "classmap": ["tests/Application/Kernel.php"],
-        "psr-4": {
-            "Tests\\SitemapPlugin\\": "tests/"
-        }
+        "sort-packages": true,
+        "bin-dir": "bin"
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.0-dev"
+            "dev-master": "1.9-dev"
```

Comment:
This should stay at `2.0-dev`, as it's related to this plugin, not to the skeleton